docs/Documentation.md

```diff
@@ -0,0 +1,128 @@
+# What's New
+* vm-manager support multiple display on Civ
+* Vm-manager monitors the state of the Civ guest to ensure that the app runs when civ is ready.
+
+# Getting Started
```

Comment:
For this part, you can refer https://github.com/projectceladon/vm_manager/wiki/User-Guide